README.rst

```diff
@@ -23,7 +23,7 @@ Or directly from git:
 
 .. code::
 
-    $ pip3 install -e "git+https://github.com/mickael9/fac.git#egg=fac-cli"
+    $ pip3 install -e "git+https://github.com/Senth/fac.git#egg=fac-cli"
```

Comment:
I'm not sure we want to do that! :smile:

Reply:
Ahh, I changed that yesterday as I released another project that used fac and wanted those installing fac to have the updated functionality (until the pull request was accepted). Didn't think the commit would be added here as well. Well, I can revert it or create a branch and submit another pull request :) No I know why you should create a branch when creating a pull request ;)

Reply:
No problem, thanks :)